src/main/java/edu/wpi/grip/core/operations/composite/FindContourOperation.java

```diff
@@ -0,0 +1,96 @@
+package edu.wpi.grip.core.operations.composite;
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+import static org.bytedeco.javacpp.opencv_core.*;
+import static org.bytedeco.javacpp.opencv_imgproc.*;
+
+/**
+ * An {@link Operation} that, given a binary image, produces a list of contours of all of the shapes in the image
+ */
+public class FindContourOperation implements Operation {
+
+    public static class Contours {
+        int rows = -1, cols = -1;
```

Comment:
Dear annoying bot,

Before you say anything, yes these are package local.  This is intentional, because they are accessed by other classes in the package (there are several operations that accept or produce contours).